osquery/tables/system/windows/system_util.cpp

```diff
@@ -89,36 +89,95 @@ std::string WmiResultItem::GetString(const std::string& name) const {
   return BSTR_to_string(value.bstrVal);
 }
 
+// Returns a pair with the first value indicating success of the query
+// and the second value being the data obtained from the query.
+std::pair<bool, bool> WmiResultItem::GetBool(const std::string& name) const {
```

Comment:
Weird, let's try to avoid this sort of result style. You can maybe return a Status with a -1, 0, or 1 result. Or yield the result bool into an output parameter.